scripts/deploy

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Is `python3` generally the name of the executable?

Reply:
In my Ubuntu, python is mapped to python2, and you need to be explicit about using python3. As a consequence, the project scripts broke on my machine.